src/nodes/Table/TableView.vue

```diff
@@ -50,25 +71,60 @@ export default {
 			type: Function,
 			required: true,
 		},
+		node: {
+			type: Object,
+			required: true,
+		},
 	},
 	data() {
 		return {
 			isEditable: false,
+			isFocused: false,
 		}
 	},
 	beforeMount() {
 		this.isEditable = this.editor.isEditable
+		this.editor.on('selectionUpdate', ({ editor }) => {
+			const startOfCurrentNode = this.getPos()
+			const endOfCurrentNode = startOfCurrentNode + this.node.nodeSize
+			const { from, to } = editor.state.selection
+			this.isFocused = from >= startOfCurrentNode && to <= endOfCurrentNode
+		})
 		this.editor.on('update', ({ editor }) => {
 			this.isEditable = editor.isEditable
 		})
 	},
+	methods: {
+		addColumnAfter() {
+			const headerRowNode = this.node.firstChild
+			this.editor.chain()
+				.focus()
+				.setTextSelection(this.getPos() + headerRowNode.nodeSize - 1)
+				.addColumnAfter()
+				.run()
+		},
+		addRowAfter() {
+			const lastRowNode = this.node.lastChild
+			this.editor.chain()
+				.focus()
+				.setTextSelection(this.getPos() + this.node?.nodeSize - lastRowNode.nodeSize + 1)
```

Comment:
The `node?` seems error-prone here. If `this.node` was faulty here, initialization of `lastRowNode` above would already fail, and it would probably lead to unwanted side-effects, no? If `this.node` can really be faulty here, I'd suggest to catch this properly and not do anything in this case.

Reply:
The node should not be null. So, I removed `?`